ush/load_obsforge_modules.sh

```diff
@@ -64,6 +64,7 @@ wxflowPATH="${HOMEobsforge}/ush/python"
 PYTHONPATH="${PYTHONPATH:+${PYTHONPATH}:}${HOMEobsforge}/ush:${wxflowPATH}:${pyiodaPATH}"
 export PYTHONPATH
 
+export PYTHONPATH="${PYTHONPATH}:${HOMEobsforge}/build/lib/python3.11/site-packages"
```

Comment:
don't force python version 3.11 here. Instead do this:
https://github.com/NOAA-EMC/global-workflow/blob/cfe3215fff72f0ea2e89c907417ed7991c3b9427/dev/ush/load_ufsda_modules.sh#L90-L106